spec/services/activities/import_from_csv_spec.rb

```diff
@@ -255,6 +259,21 @@
       expect(subject.errors.first.message).to eq(I18n.t("importer.errors.activity.invalid_intended_beneficiaries"))
     end
 
+    it "has an error if the GDI is invalid" do
+      new_activity_attributes["GDI"] = "Here is a string"
```

Comment:
When you say "we're given a code, not a string" - the code is still presented as a string, right?

Reply:
Oh yeah, this is just me putting any old rubbish in the field. I'll change it out, so it's clearer.